Week1/homework/app.js

```diff
@@ -1,11 +1,84 @@
-'use strict';
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+'use strict'; {
+  const myBooks = [
+    'angels_&_demons',
+    'the_da_vinci_code',
+    'anna_of_the_five_towns',
+    'harry_potter_chamber_of_secrets',
+    'the_god_delusion',
+    'the_old_man_and_the_sea',
+    'righteous',
+    'the_fourth_monkey',
+    'norse_mythology',
+    'lincoln_in_the_bardo'
   ];
 
+  function main() {
+    const root = document.getElementById('root');
+    const ul = document.createElement('ul');
+    root.appendChild(ul);
+    myBooksDetails(ul);
+    imagesAndStyling();
+  }
 
-  // Replace with your own code
-  console.log(bookTitles);
+  const booksDetails = {
+    'angels_&_demons': { title: 'Angels & Demons', author: 'Dan Brown', language: 'English' },
+    'the_da_vinci_code': { title: 'The DaVinci Code', author: 'Dan Brown', language: 'English' },
+    'anna_of_the_five_towns': { title: 'Anna Of The Five Towns', author: 'Arnold Bennett', language: 'English' },
+    'harry_potter_chamber_of_secrets': { title: 'Harry Potter Chamber Of Secrets', author: ' J. K. Rowling', language: 'English' },
+    'the_god_delusion': { title: 'The God Delusion', author: 'Richard Dawkins', language: 'English' },
+    'the_old_man_and_the_sea': { title: 'The Old Man And The Sea', author: 'Ernest Hemingway', language: 'English' },
+    'righteous': { title: 'Righteous', author: 'Joe Ide', language: 'English' },
+    'the_fourth_monkey': { title: 'The Fourth Monkey', author: 'Jeffery Deaver', language: 'English' },
+    'norse_mythology': { title: 'Norse Mythology', author: 'Neil Gaiman', language: 'English' },
+    'lincoln_in_the_bardo': { title: 'Lincoln In The Bardo', author: 'George Saunders', language: 'English' }
+  };
+
+
+  function myBooksDetails(ul) {
```

Comment:
For function names, alway try to give a name that indicates some sort of action, because functions 'do' something. So the name should have a verb as its head word. For instance, this function could be called `renderBookDetails()`.

Reply:
I will keep that in mind next time.